### PR TITLE
Run prettier, convert to standard eslint, add CI

### PR DESCRIPTION
**Pull request by hipstersmoothie**

### CODE REVIEW

File: lib/github.js [hunk 1]
```diff
@@ -1,39 +1,41 @@
 /* eslint-disable camelcase */
 
-import { log, wasReported } from './log.js';
-import assert from 'assert';
-import fs from 'fs';
-import progress from 'request-progress';
-import request from 'request';
+import assert from "assert";
```
Comment (leerob) [inline lib/github.js:3]: Did we change the Prettier confi away from single quotes?

Comment (leerob) [reply lib/github.js:3]: Nvm, that's in `pkg`. Let's use the same config.

File: package.json [hunk 2]
```diff
@@ -43,30 +43,32 @@
...
   "scripts": {
-    "babel": "node test/rimraf-es5.js && babel lib --out-dir lib-es5",
+    "build": "node test/rimraf-es5.js && babel lib --out-dir lib-es5",
     "bin": "node lib-es5/bin.js",
-    "lint": "eslint-klopov . || true",
-    "prepare": "npm run babel",
-    "prepublishOnly": "eslint-klopov . && npm test",
+    "lint": "eslint lib || true",
+    "prepare": "npm run build",
+    "prepublishOnly": "npm run lint && npm test",
```
Comment (leerob) [inline package.json:68]:
```suggestion
    "prepublishOnly": "npm run lint",
```

File: lib/github.js (hunk 1 above)
Comment (hipstersmoothie) [reply lib/github.js:3]: Resolved!

**Pull request merged**